### PR TITLE
fix serial issue on stable 1.3

### PR DESCRIPTION
**Pull request by AsuraZeng**

### CODE REVIEW

**Comment by BaochengSu** [comment]
Please mark which commit is backported from master.

For the other commit, please also commit it to master.

**Comment by AsuraZeng** [comment]
> 
Add messages to the commit body already,  pls check.

File: recipes-core/customizations-example/files/board-configuration [hunk 1]
```diff
@@ -69,7 +69,7 @@ def initAruinoPins():
 
 
 def initExternalSerialMode():
-    command = 'grep -a -o -P "IOT2050 \w*" /proc/device-tree/model'
+    command = 'grep -a -o -P "IOT2050 [\w\s]+" /proc/device-tree/model'
```
Comment (BaochengSu) [inline recipes-core/customizations-example/files/board-configuration:72]: Why need change this?

Comment (AsuraZeng) [reply recipes-core/customizations-example/files/board-configuration:72]: This is intended to distinguish between `Advanced`  and `Advanced PG2`.
and do the  the serial port reset for PG2 devices

**Review by BaochengSu — APPROVED** [review]

**Comment by jan-kiszka** [comment]
What speaks against picking complete https://github.com/siemens/meta-iot2050/pull/369 for stable?

Ack for the second patch already.

**Comment by AsuraZeng** [comment]
> What speaks against picking complete #369 for stable?
> 
> Ack for the second patch already.

Not fully tested  for the #369

**Review by jan-kiszka — CHANGES_REQUESTED** [review]
Please rebase over the CI fix. Then it looks like we need to move forward with this stable-only fix for the serial config.

**Review by jan-kiszka — APPROVED** [review]

**Comment by BaochengSu** [comment]
Merged and closed

**Pull request closed without merging**